tests/utils/next-version-helpers.mjs

```diff
@@ -6,7 +6,7 @@ import fg from 'fast-glob'
 import { coerce, gt, gte, satisfies, valid } from 'semver'
 import { execaCommand } from 'execa'
 
-const FUTURE_NEXT_PATCH_VERSION = '14.999.0'
+const FUTURE_NEXT_PATCH_VERSION = '15.999.0'
```

Comment:
unrelated to rest - it just impacts some conditional assertions depending on next versions when running tests locally and not setting `RESOLVED_NEXT_VERSION` env var (like CI does) - the idea of this const is to always use latest version conditionals when running locally as this is most likely what you'd test with